source/translator.d

```diff
@@ -0,0 +1,35 @@
+private string[string] defaultLanguage;
+static this()
+{
+	defaultLanguage = [
+		"editor.run": "Run",
+		"editor.reset": "Reset",
+		"editor.format": "Format",
+		"editor.keyboard_shortcuts": "Keyboard Shortcuts",
+	];
+}
+
+/**
+Allows per-language overwrites of UI text
+*/
+class Translator
```

Comment:
It being a `class` has the the advantage of having automatic by-reference copy for the diet templates (on every request), so imho there's already a need for it.